src/app/components/price-details/price-details.component.scss

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap');
```

Comment:
No need to import, there is a link in index.html that laods the fonts to the project.